tests/installation/start_install.pm

```diff
@@ -80,7 +80,7 @@ sub run {
         assert_screen("inst-packageinstallationstarted", $started_timeout);
     }
     else {
-        wait_still_screen(3);    # wait so alt-i is pressed when installation overview is not being generated
+        wait_still_screen(30);    # wait so alt-i is pressed when installation overview is not being generated
```

Comment:
So now every installation will wait 30 seconds, that is not ok.

Reply:
I changed the time to 10 and added is_ipmi condition.